common/topology/src/lib.rs

```diff
@@ -176,6 +176,18 @@ impl NymRouteProvider {
             .random_route_to_egress(rng, egress_identity, self.ignore_egress_epoch_roles)
     }
 
+    /// Tries to create a route to the egress point, such that it goes through mixnode on layer 1,
```

Comment:
I don't think this comment is accurate as you're not actually going through any mixnodes. just straight into egress

Reply:
Fixed